redash/handlers/dashboards.py

```diff
@@ -25,6 +41,11 @@ class DashboardListResource(BaseResource):
     def get(self):
         dashboards = [d.to_dict() for d in models.Dashboard.all(self.current_org, self.current_user.groups, self.current_user)]
 
+        for dashboard in dashboards:
+            last_change = models.Change.get_latest(object_id=dashboard['id'], object_type=models.Dashboard.__name__)
+            if last_change:
+                dashboard['latest_version'] = last_change.id
```

Comment:
Done. This change also impacted other parts of the code (e.g., the UI code where we previously tracked a propery `latest_version` which is now simply the `version` property of the model instance).

We've re-run all tests and did additional manual testing to ensure that the original functionality is still preserved.